index.html

```diff
@@ -8,7 +8,7 @@
     <style>
```

Comment:
hey, move styles to file!

Reply:
You do it. I'm too lazy.

Reply:
No problema, I proceed to move styles to file, after merge. You Will pay me at lunch.

Reply:
Get a job mate.